docs/install_occlum_packages.md

```diff
@@ -0,0 +1,99 @@
+# Install Occlum Packages with Package Mangement Tool
+
+Occlum packages can be easily installed and deployed using mainstream package management tool including RPM and APT.
+
+
+### Prerequisite
+
+**Install enable_RDFSBASE Kernel Module**
+
+Please follow [this README](https://github.com/occlum/enable_rdfsbase/blob/master/README.md) to install `enable_rdfsbase` kernel module.
+
+
+### Install Occlum with RPM
+
+1. Install Prerequisite
+```
+yum install -y wget yum-utils make jq gdb
+```
+
+2. Install Intel® SGX Driver and Intel® SGX PSW
+
+Please follow [Intel SGX Installation Guide](https://download.01.org/intel-sgx/sgx-linux/2.11/docs/Intel_SGX_Installation_Guide_Linux_2.11_Open_Source.pdf) to install SGX driver and SGX PSW. SGX SDK is not required. Using RPM installer is recommanded. 
```

Comment:
Should we refer the doc or we can just extract the commands from the doc to save our readers from looking into the doc themselves? The latter one seems more user friendly to me. I think it is acceptable since Occlum has some requirements on the version of Intel SGX PSW. So the installation steps of that version of SGX PSW won't be updated by Intel in the future and we can write them down here.

Reply:
Besides PSW, there is also SGX driver. I agree that put installation guide here can be convenient for users. However, installing driver and PSW is not that easy. And users can meet all kinds of problems. I would hand in the professional stuff to professional people. Thus, I choose to refer to the original documents. I think PSW version should be forward compatible and thus should not cause problems.